Fix cloud build deployment of master branch

ramyabuva opened the pull request:
On the cloudbuild, there was an issue building the docker container, so instead i replaced it with a different tagged image and removed the dockerfile.

wli commented:
What was the problem + fix? Can you write it in the PR description so it's clear?

Also, why are there two cloud build checks, and only one ran?

![image](https://user-images.githubusercontent.com/649892/87965864-48d11a80-ca71-11ea-83c3-20d921e78683.png)

ramyabuva commented:
> What was the problem + fix? Can you write it in the PR description so it's clear?
> 
> Also, why are there two cloud build checks, and only one ran?
> 
> ![image](https://user-images.githubusercontent.com/649892/87965864-48d11a80-ca71-11ea-83c3-20d921e78683.png)

Just updated the PR description. The 2nd test is a trigger I created on cloudbuild to see if deployment worked, however I deleted it before making this PR, which is why the check won't run.

wli commented on cloudbuildMaster.yaml:
```diff
@@ -1,5 +1,4 @@
-   args: ['deploy']
+ - name: maven:3-jdk-8
+   entrypoint: mvn
+   args: ['package', 'appengine:deploy']
```
nit: missing newline
